src/MovieDetails/MovieDetails.scss

```diff
@@ -27,7 +27,7 @@
 .overlay {
   position: relative;
   z-index: 2;
-  height: 43.5rem;
+  height: 80vh;
   width: 100%;
   /* Permalink - use to edit and share this gradient: https://colorzilla.com/gradient-editor/#22262b+0,7db9e8+100&1+0,0+100 */
```

Comment:
We should be sure to put this link within the README

Reply:
Good idea! I'll delete the link and put it in the README. It was such a great resource!